src/reducers/index.js

```diff
@@ -13,8 +13,10 @@ export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
     // Fill in the body of this case
+      return {...state, count: state.count+1}
```

Comment:
I would also say that this spread is unnecessary since you only have one key in your state object and you are providing that value directly. The spread operator would make sense if you had many different keys in your state object and only wanted to modify one of those keys. 

For example:

```js
let state = {
    count: 0,
    anotherKey: 'idk'
}

return { ... state, count: state.count + 1 }

```